connector/grafanacloudconnector/host_metrics.go

```diff
@@ -0,0 +1,72 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package grafanacloudconnector // import "github.com/open-telemetry/opentelemetry-collector-contrib/connector/grafanacloudconnector"
+
+import (
+	"sync"
+	"time"
+
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+)
+
+type hostMetrics struct {
+	hosts map[string]struct{}
+	mutex sync.RWMutex
```

Comment:
nit: could we revert the order of these two fields so if we ever add more, it's easy to group which fields mutex protects?

Reply:
done 👍